- admin panel i18n

kamilpiech97 opened the pull request.

EwelinaSkrzypacz reviewed (commented):
1. This translation sounds a bit awkward

![image](https://github.com/blumilksoftware/website/assets/56546832/a509be0d-293e-460e-b0fa-e3ae4ae4e523)
 
 2. "Published from" and "published to" is not translated
 
 
![image](https://github.com/blumilksoftware/website/assets/56546832/6fceeb7a-42cf-47b7-aafa-048fa56c992c)


3. Alb\so format date is bad - we should have dd.mm.yyyy (now we have mm/dd/yyyy)

and calendar is also in English

kamilpiech97 commented:
> 1. This translation sounds a bit awkward
> 
> ![image](https://private-user-images.githubusercontent.com/56546832/323888873-a509be0d-293e-460e-b0fa-e3ae4ae4e523.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM1Mjk0MzMsIm5iZiI6MTcxMzUyOTEzMywicGF0aCI6Ii81NjU0NjgzMi8zMjM4ODg4NzMtYTUwOWJlMGQtMjkzZS00NjBlLWIwZmEtZTNhZTRhZTRlNTIzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE5VDEyMTg1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIwNTYxODdmMzIyMWZlNWM3MDVhNGI5MmVlZTk4OGRlMmE1N2QzZTAxYzIxNWVmYTc2NGNhMjMyMWY2OTgyOGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.86zW07ZwdEwjNVCRGPRbnPM51fCkuv5EUOXHhHFjiJk)
> 
> 2. "Published from" and "published to" is not translated
> 
> ![image](https://private-user-images.githubusercontent.com/56546832/323890809-6fceeb7a-42cf-47b7-aafa-048fa56c992c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM1Mjk0MzMsIm5iZiI6MTcxMzUyOTEzMywicGF0aCI6Ii81NjU0NjgzMi8zMjM4OTA4MDktNmZjZWViN2EtNDJjZi00N2I3LWFhZmEtMDQ4ZmE1NmM5OTJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE5VDEyMTg1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVkZGUyN2VkN2IzNTE5ZmMyMjNlOTY5ZmU1NTY5MjBkY2MxNzJlNzU5ZmI0MmE2YjliYzM0ZjQ0ZDM0MDIyYWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zGtTnWCVh1W8uxitE-HkisgUtbdlhj24LwEXE3xMLSM)
> 
> 3. Alb\so format date is bad - we should have dd.mm.yyyy (now we have mm/dd/yyyy)
> 
> and calendar is also in English

1. changed rule
2. translated
3. changed

EwelinaSkrzypacz reviewed (commented):
1.  When I fill form on create activity page like on screenshot below (title, subtitle, description, but only in Polish)  I see error

![image](https://github.com/blumilksoftware/website/assets/56546832/bb87dd06-dbcd-40db-a764-d0c9072ac83a)

2. I have still calendar in English (and date format) - I have Chrome in English and I see formatted date mm/dd/yyyy, but on Firefox in Polish I have format like dd . mm . rrrr (and yes, with spaces between dots)

![image](https://github.com/blumilksoftware/website/assets/56546832/c70719c9-3b2e-4bac-9ec7-acc4f3cbb7ad)


![image](https://github.com/blumilksoftware/website/assets/56546832/d4b4f06b-0277-4f2f-87bd-7d84b3f3a71b)

EwelinaSkrzypacz reviewed (approved):
I give approve, because dates works - it's browser fault.
